azext_iot/iothub/providers/state.py

```diff
@@ -0,0 +1,310 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+
+from azext_iot.common.shared import DeviceAuthApiType, DeviceAuthType, ConfigType
+from azext_iot.iothub.providers.base import IoTHubProvider
+from azext_iot.operations.hub import _iot_device_show, _iot_device_module_twin_show, _iot_device_module_create, \
```

Comment:
you can use () here too, look at other files that have many imports from a single module

Reply:
is it better practice to then put each import on a new line? I see that's what most of the other files are doing, my only reservation is that... that would be a whole lot of lines

Reply:
yeah but it is better practice - plus most editors will let you collapse things within a block/grouped in parentheses